### ISSUE TITLE
When an active app (not killed by iOS or manually closed) is opened, the animation slides in from the right.

### ISSUE THREAD

**Issue by dallaslittle** [LaughingQuoll/NCXI#24]
This is the most consistent bug I've found. If this could be solved, it would take this tweak a huge step forward. It ruins the OS animations as a whole.

**Comment by LaughingQuoll** [comment]
Does the latest deb (in packages folder) do this? I can't recreate it on any device I own.

**Comment by dallaslittle** [comment]
Yes, it does. At first, apps open normally. Then slowly, they all start opening from the right side of the screen, until it's basically one of the only animations you see. 

For what it's worth, I've had NCXI installed with only 3-4 tweaks at one point and it was still doing it. 

This was after a fresh restore as well.
